metrics/metrics_event_publisher.go

```diff
@@ -82,26 +82,26 @@ func enabledGauge() prometheus.Gauge {
 	return enabledGauge
 }
 
-func blacklistGauge() *prometheus.GaugeVec {
-	blacklistCnt := prometheus.NewGaugeVec(
+func denylistGauge() *prometheus.GaugeVec {
+	denylistCnt := prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "blocky_blacklist_cache",
-			Help: "Number of entries in the blacklist cache",
+			Name: "blocky_denylist_cache",
+			Help: "Number of entries in the denylist cache",
 		}, []string{"group"},
 	)
 
-	return blacklistCnt
+	return denylistCnt
 }
 
-func whitelistGauge() *prometheus.GaugeVec {
-	whitelistCnt := prometheus.NewGaugeVec(
+func allowlistGauge() *prometheus.GaugeVec {
+	allowlistCnt := prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "blocky_whitelist_cache",
-			Help: "Number of entries in the whitelist cache",
+			Name: "blocky_allowlist_cache",
```

Comment:
renaming the metric will break existing grafana dashboards. Better here: create new metric with new name and mark "old" as deprecated. Both metrics should count the same value